griptape/utils/observability/griptape_cloud_observability.py

```diff
@@ -0,0 +1,59 @@
+import atexit
+import os
+
+from griptape.utils.observability.griptape_cloud_span_exporter import GriptapeCloudSpanExporter
+from griptape.utils.observability.griptape_instrumentor import GriptapeInstrumentor
+from opentelemetry import trace
+from opentelemetry.trace import Status, StatusCode
+from opentelemetry.instrumentation.threading import ThreadingInstrumentor
+from opentelemetry.sdk.resources import Resource
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+
+
+class GriptapeCloudObservability:
+    def __init__(self):
+        self.trace_provider = None
+        self.root_span_context_manager = None
+        self.exit_handler = None
+
+    def start(self):
+        # 1. Configure griptape exporter
+        resource = Resource(attributes={"service.name": "unknown"}, schema_url=None)
+        self.trace_provider = TracerProvider(resource=resource)
+        trace.set_tracer_provider(self.trace_provider)
+        self.trace_provider.add_span_processor(BatchSpanProcessor(GriptapeCloudSpanExporter()))
```

Comment:
What if users want to swap out their own exporter?

Reply:
Refactored into a driver to allow switching implementations.